### PR TITLE
SLVS-1199 Add SLCoreAnalyzer, disable SecretsAnalyzer

### PR DESCRIPTION
**Pull request by georgii-borovinskikh-sonarsource**

### CODE REVIEW

File: src/CloudSecrets/SecretsAnalyzer.cs
Comment (ugras-ergun-sonarsource) [inline src/CloudSecrets/SecretsAnalyzer.cs:1]: why not just delete the file?

File: src/SLCore.UnitTests/Analysis/SLCoreAnalyzerTests.cs [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        var timestampTestAssert = DateTimeOffset.Now.ToUnixTimeMilliseconds();
+        analysisService.Received().AnalyzeFilesAndTrackAsync(Arg.Is<AnalyzeFilesAndTrackParams>(a => 
+                a.analysisId == analysisId 
+                && a.configurationScopeId == "someconfigscopeid"
+                && a.filesToAnalyze.Single() == new FileUri(@"C:\file\path")
+                && a.extraProperties != null
+                && a.shouldFetchServerIssues
+                && a.startTime >= timestampTestStart
+                && a.startTime <= timestampTestAssert),
+            Arg.Any<CancellationToken>());
```
Comment (ugras-ergun-sonarsource) [inline src/SLCore.UnitTests/Analysis/SLCoreAnalyzerTests.cs:127]: Pass a cancellation token to ExecuteAnalysis and check if it is propagated here.

**Review by ugras-ergun-sonarsource — COMMENTED** [review]
some comments

**Review by ugras-ergun-sonarsource — APPROVED** [review]

File: src/SLCore.UnitTests/Analysis/SLCoreAnalyzerTests.cs (hunk 1 above)
Comment (georgii-borovinskikh-sonarsource) [reply src/SLCore.UnitTests/Analysis/SLCoreAnalyzerTests.cs:127]: The test is below

File: src/CloudSecrets/SecretsAnalyzer.cs
Comment (georgii-borovinskikh-sonarsource) [reply src/CloudSecrets/SecretsAnalyzer.cs:1]: I left it for this task https://sonarsource.atlassian.net/browse/SLVS-1191

**Pull request merged**